dev/scripts/exglobal_enkf_earc_tars.py

```diff
@@ -28,7 +28,8 @@ def main():
             'DOHYBVAR', 'DOIAU_ENKF', 'IAU_OFFSET', 'DOIAU', 'DO_CA',
             'DO_CALC_INCREMENT', 'assim_freq', 'ARCH_CYC', 'DO_JEDISNOWDA',
             'ARCH_WARMICFREQ', 'ARCH_FCSTICFREQ', 'DOHYBVAR_OCN',
-            'DOLETKF_OCN', 'IAUFHRS_ENKF', 'NET', 'NMEM_ENS_GFS', 'DO_GSISOILDA', 'DO_LAND_IAU']
+            'DOLETKF_OCN', 'IAUFHRS_ENKF', 'NET', 'NMEM_ENS_GFS', 'DO_GSISOILDA', 'DO_LAND_IAU',
+            'DOENKFONLY_ATM']
```

Comment:
Cleans up the format of this block better
```suggestion
            'DOLETKF_OCN', 'IAUFHRS_ENKF', 'NET', 'NMEM_ENS_GFS',
            'DO_GSISOILDA', 'DO_LAND_IAU', 'DOENKFONLY_ATM']
```

Reply:
@TravisElless-NOAA Will modify as suggested in the updated PR. Thanks.